new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleNginxValidation.java

```diff
@@ -628,4 +673,149 @@ private void checkServiceCreated(String serviceName) {
 
   }
 
+  /**
+   * Check pod does not exist.
+   *
+   * @param podName pod name to check
+   */
+  private void checkPodDoesNotExist(String podName) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pod {0} to be removed in namespace {1} "
```

Comment:
method name message does not match. if the method simply checks if a pod exists or not why is it waiting here ?

Reply:
changed the method name to checkPodDeleted(String podName)